modules/registry-configuring-storage-gcp-user-infra.adoc

```diff
@@ -18,9 +18,33 @@ cloud credentials.
 contents of a credentials file provided by GCP:
 ** `REGISTRY_STORAGE_GCS_KEYFILE`
 
-
 .Procedure
 
-////
-Stub for procedure to manually set storage medium.
-////
+Use the following procedure if the Registry Operator cannot create a GCP bucket
```

Comment:
I think this sentence would sit better after the first sentence in the module or even combined with the first sentence, so a customer knows immediately that this module might or might not address their needs. How about:

> If the Registry Operator cannot create a Google Cloud Platform (GCP) bucket, you must set up the storage medium manually and configure the settings in the
registry custom resource (CR).